MetaMask/metamask-extension
# feat: add tracking event to Receive button

## **Description**

Adds tracking event to the open Receive Modal trigger

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/26958?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/3202

## **Manual testing steps**

1. Open Receive Modal from the new blue button on home
2. Check that the trackEvent gets triggered (coin-buttons.tsx)

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
